### ISSUE TITLE
GIMP file couldn't uploaded

### ISSUE THREAD

**Issue by ChrissW-R1** [RocketChat/Rocket.Chat#14120]
### Description:

File created by GIMP (file extension *.xcf) couldn't be uploaded.

### Steps to reproduce:

1. Add "image/*" (default) or "image/x-xcf" (exact MIME type) to the list of "Accepted Media Types" (Administration --> File Upload)
2. Open a channel
3. Try to upload a file produced with GIMP

### Expected behavior:

The file will be uploaded and a message with the attachment will be posted to the channel.

### Actual behavior:

The upload will be rejected with the note "Media Types Not Accepted".
![image](https://user-images.githubusercontent.com/1086210/56037098-d2eae900-5d2e-11e9-8abd-ca5676b7035e.png)

### Server Setup Information:

- Version of Rocket.Chat Server: 0.74.3
- Operating System: CentOS 7
- Deployment Method: manually
- Number of Running Instances: 1
- DB Replicaset Oplog: Disabled
- NodeJS Version: 8.11.3
- MongoDB Version: 3.6.12

**Comment by lolimay** [comment]
The `*.xcf` files is the source file of an image (like `*.psd`). In fact, there's no need to support this format. You can simply export your image to `*.jpg/*.png` or other common image formats by <kbd>Ctrl</kbd> + <kbd>E</kbd>. Hope it can help you :)

**Comment by ChrissW-R1** [comment]
No, there is a huge need, because we need to exchange the source files for working on them.

**Comment by ChrissW-R1** [comment]
We have the same issue with text files, that don't have the *.txt extension.
For example there is a MatLab file (*.m) which have the MIME type text/plain. This MIME type is in the granted list, but it is not possible to upload them.

**Comment by foss-** [comment]
Same is true for .pgp .gpg files. Would be nice to add those to the list as well.
